_reports/virresh.md

```diff
@@ -0,0 +1,391 @@
+      - details: >
+          ".coafile: Add PyDocStyleBear"
+  - 31:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/1a348b4e
+      - details: >
+          ".gitlab-ci.yml: Add git installation"
+  - 32:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/ba4adcdb
+      - details: >
+          "coantlib: Generalise entrypoint"
+  - 33:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/72e2bb72
+      - details: >
+          "tests: Remove hashbangs"
+  - 34:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/abc9f3b0
+      - details: >
+          "Python3.py: Make parser PEP8 compliant"
+  - 35:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/bfa10282
+      - details: >
+          "ASTBear: Remove redundant `lang` argument"
+  - 36:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/27891c28
+      - details: >
+          "XMLIndentBear: Add XMLIndentBear"
+  - 37:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/a8a7f6eb
+      - details: >
+          ".ci: Add check for XML Upstream grammar"
+  - 38:
+      - repo: coala
+      - link: https://github.com/coala/coala/commit/d936d162
+      - details: >
+          "XML.py: Add language definition for XML"
+  - 39:
+      - repo: community
+      - link: https://github.com/coala/community/commit/7e5cd645
+      - details: >
+          ".coafile: Add PyPluralNamingBear"
+  - 40:
+      - repo: coala-quickstart
+      - link: https://github.com/coala/coala-quickstart/commit/bf587786
+      - details: >
+          "appveyor.yml: Use pip as a module"
+  - 41:
+      - repo: coala-quickstart
+      - link: https://github.com/coala/coala-quickstart/commit/99ad0910
+      - details: >
+          "test-requirements.txt: Add packaging"
+  - 42:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/97e98900
+      - details: >
+          "GitCommitBear: Refactor issue regexes"
+  - 43:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/33cd9e41
+      - details: >
+          "GitCommitBear: Add BitBucket type issue references"
+  - 44:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/985217dc
+      - details: >
+          "HgCommitBear: Switch to git-url-parse"
+  - 45:
+      - repo: projects
+      - link: https://github.com/coala/projects/commit/a8f0e9b0
+      - details: >
+          "report.md: Make GSoC year dynamic"
+  - 46:
+      - repo: coala-mobans
+      - link: https://gitlab.com/coala/mobans/commit/76ebb40c
+      - details: >
+          "appveyor.yml.jj2: Add template for appveyor config"
+  - 47:
+      - repo: grammars-v4
+      - link: https://github.com/antlr/grammars-v4/pull/1205
```

Comment:
try to get this merged : https://github.com/antlr/grammars-v4/pull/1205#issuecomment-411577709

if it is an unmerged PR, move it to last in the list